### PR TITLE
Fix/organize

### PR DESCRIPTION
**Pull request by TidalPaladin**
Includes changes to the data organization pipeline to support new data

### CODE REVIEW

File: dicom_utils/container/output.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def write(self, collection: RecordCollection, dest: Path) -> RecordCollection:
         return result
 
 
-@OUTPUT_REGISTRY(name="longitudinal", subdir="cases", derived=True)
+# @OUTPUT_REGISTRY(name="longitudinal", subdir="cases", derived=True)
```
Comment (timcogan) [inline dicom_utils/container/output.py:107]: Should this be removed?

File: dicom_utils/container/output.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def __call__(self, inp: Union[Input, Dict[str, RecordCollection]]) -> Dict[str,
             if self.by_case:
                 f.write(f"{name}\n")
             else:
-                files = [Path(*rec.path.parts[-2:]) for rec in collection]
+                files = [Path(*rec.path.parts[-4:]) for rec in collection]
```
Comment (timcogan) [inline dicom_utils/container/output.py:207]: Having a named constant for this `-4` would be helpful for interpret-ability.

File: dicom_utils/container/output.py (hunk 1 above)
Comment (TidalPaladin) [reply dicom_utils/container/output.py]: I added a remark that this output handler was how we originally handled longitudinal data and that a new organizational system has replaced it. I don't see the need to keep it registered if we plan to remove it in the future.

File: dicom_utils/container/output.py (hunk 2 above)
Comment (TidalPaladin) [reply dicom_utils/container/output.py]: Moved to a new function with explanation

**Review by timcogan — APPROVED** [review]

**Pull request merged**